src/common/image.c

```diff
@@ -477,7 +477,7 @@ static void _set_datetime(const int32_t imgid, const char *datetime)
   /* fetch image from cache */
   dt_image_t *image = dt_image_cache_get(darktable.image_cache, imgid, 'w');
 
-  memcpy(&image->exif_datetime_taken, datetime, sizeof(image->exif_datetime_taken));
+  g_strlcpy(&image->exif_datetime_taken, datetime, sizeof(image->exif_datetime_taken));
```

Comment:
seems like CI doesn't like that one ;)

```
fatal error: incompatible pointer types passing 'char (*)[20]' to parameter of type 'gchar *' (aka 'char *') [-Wincompatible-pointer-types]
```

Reply:
Actually exposed a bug that was there all the time, shouldn't be an addr, it's already a char array...